scout/server/blueprints/variants/templates/variants/sv-variant.html

```diff
@@ -223,13 +228,24 @@
 
   <div class="row">
     <div class="col-md-12">
-      {{ genes(variant) }}
+      {{ overlapping(overlapping_snvs, variant.rank_score) }}
     </div>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
-      {{ overlapping(overlapping_snvs, variant.rank_score) }}
+    <div class="col-12">
+      <div class="card">
+        <nav>
+          <div class="nav nav-tabs" id="nav-tab" role="tablist">
+            <a class="nav-item nav-link active" id="nav-genes-tab" data-toggle="tab" href="#nav-genes" role="tab" aria-controls="nav-genes" aria-selected="true">Genes</a>
+            <a class="nav-item nav-link" id="nav-transcripts-tab" data-toggle="tab" href="#nav-transcripts" role="tab" aria-controls="nav-transcripts" aria-selected="false">Transcripts</a>
+          </div>
+        </nav>
+        <div class="tab-content" id="nav-tabContent">
+          <div class="tab-pane fade show active" id="nav-genes" role="tabpanel" aria-labelledby="nav-genes-tab">{{ genes_panel(variant) }}</div>
```

Comment:
There's a difference in the styles of the 2 tabs: genes_panel and transcript_panel macros:

![image](https://user-images.githubusercontent.com/28093618/66653720-f4268680-ec38-11e9-8c2f-e511b2db3984.png)

This reflects in 2 different behaviors of the tabs:

![image](https://user-images.githubusercontent.com/28093618/66653765-09031a00-ec39-11e9-9973-c39e399552ec.png)


And 

![image](https://user-images.githubusercontent.com/28093618/66653787-14eedc00-ec39-11e9-998b-10d59af5af03.png)


You could maybe adopt the same style for both?